README.md

```diff
@@ -459,6 +565,7 @@ is the results of the metric. Some metrics are computed for multiple values.
 | `erf` | Labels | ERF | 0.10 |
 | `atd` | Average time to discovery (in label actions) | - | - |
 | `td` | Row number (starting at 0) | Number of records labeled | - |
+| `cm` | Recall  | Confusion matrix values at recall | 0.95, 1 |
 
```

Comment:
I now show the 0.95 and 1 as default intercepts in sample output, however this does not align with current code
Another small issue is that with the command --cm 0.95 1, TNR is outputted at those intercepts as well.

Reply:
Oke, I see your issue regarding readability. This should be resolved later with some API changes.